packages/airnode-node/src/evm/verification/request-verification.ts

```diff
@@ -16,32 +16,34 @@ export function verifySponsorWallets<T>(
   requests: Request<T>[],
   masterHDNode: ethers.utils.HDNode
 ): LogsData<Request<T>[]> {
-  const logsWithVerifiedRequests: LogsData<Request<T>>[] = requests.map((request) => {
+  const logs: PendingLog[] = [];
+  const verifiedRequests: Request<T>[] = [];
+
+  requests.forEach(function (request) {
     if (request.status !== RequestStatus.Pending) {
```

Comment:
This will have to go eventually as well.

Reply:
Yes indeed 😅